### PR TITLE
support CSS modules

### LINKED ISSUE #3: usage with css modules
Feels a little less than ideal. Maybe worth adding in a special case or another function where you can pass in just a class name.

``` coffeescript
classNames = require './index.styl'
e = require 'react-e'

#...

  render: ->
    e ".#{classNames.root}",
      e ".#{classNames.title}", 'My Todo List'
      e ".#{classNames.links}",
        # ...
```

### PR DESCRIPTION
**Pull request by charlierudolph**
@alexdavid 

resolves #3 and #5

### CODE REVIEW

File: bind.js [hunk 1]
```diff
@@ -0,0 +1 @@
+require('./lib/bind.js');
```
Comment (alexdavid) [inline bind.js]: shouldn't this be `module.exports = require('./lib/bind.js');`?

Comment (charlierudolph) [reply bind.js]: Yes indeed

File: README.md [hunk 2]
```diff
@@ -21,3 +21,27 @@ Is equivalent to:
...
 </div>
 ```
+
+## Use with css modules
+
+```coffeescript
+styles = require 'index.css'
+###
+styles =
+  container: 'local1
```
Comment (alexdavid) [inline README.md]: missing a quote here

**Comment by charlierudolph** [comment]
@alexdavid good to merge and release a new version (which I think needs to be major since react moved from a dependency to a peer dependency)?

**Review by alexdavid — APPROVED** [review]
Yeah I think this is good to go

My only question is that in the issue you mentioned that you wanted to throw errors if a class didn't exist in the class list. Did you change your mind on that?

**Comment by charlierudolph** [comment]
We are delegating to classNames which doesn't have that functionality, and I don't see an easy way to implement it

**Comment by alexdavid** [comment]
Good point.

🚀

**Pull request closed without merging**